server.go

```diff
@@ -782,6 +785,18 @@ func (sp *serverPeer) enforceNodeBloomFlag(cmd string) bool {
 	return true
 }
 
+func (sp *serverPeer) OnFeeFilter(p *peer.Peer, msg *wire.MsgFeeFilter) {
+	// Check that the passed minimum fee is a valid amount.
+	if msg.MinFee < 0 || msg.MinFee > btcutil.MaxSatoshi {
+		peerLog.Debugf("Peer %v sent an invalid feefilter '%v' -- "+
+			"disconnecting", sp, msg.MinFee)
```

Comment:
I would suggest this should use ` btcutil.Amount(msg.MinFee)` so that the log will have it nicely formatted with the unit as opposed to just being a raw number.